### ISSUE TITLE
App crashes on iOS 13.3.1

### ISSUE THREAD

**Issue by ClayMav** [stripe/stripe-terminal-ios#72]
## Summary
When running the Terminal app on iOS 13.3, everything works as expected. However, a change occurred in version 13.3.1 which produces the below error:
```
dyld: Library not loaded: @rpath/Static.framework/Static
  Referenced from: /private/var/containers/Bundle/Application/9E01462D-1C2A-4C85-BF7F-571A8F79524C/Example.app/Example
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/9E01462D-1C2A-4C85-BF7F-571A8F79524C/Example.app/Frameworks/Static.framework/Static: code signature invalid for '/private/var/containers/Bundle/Application/9E01462D-1C2A-4C85-BF7F-571A8F79524C/Example.app/Frameworks/Static.framework/Static'

(lldb) 
```
In my investigations for a fix, I stumbled across a thread confirming the issue is not exclusive to this app.
[https://github.com/Alamofire/Alamofire/issues/3051](https://github.com/Alamofire/Alamofire/issues/3051
)

## Code to reproduce
No changes required from master to reproduce.
Ran the setup.sh before I was able to install it on the device due to build failure.

## iOS version
13.3.1

## Installation method

Just running the example app.

## SDK version

In the example app.

## Other information

None.

**Comment by jil-stripe** [comment]
@ClayMav, sorry this is happening! Looks like Apple [changed a setting so that external frameworks don't link properly on 13.1.1 for free accounts](https://stackoverflow.com/questions/60096258/library-not-loaded-rpath-fblpromises-framework-fblpromises-ios-13-3-1/60211703#60211703). I'll work on patching the example app to work around this limitation.

Could you let me know if you have a free or paid Apple Developer account, so I can narrow down whether you're experiencing this issue or something else?

I'll respond here if I have any updates. Thanks for reporting.
James

**Comment by ClayMav** [comment]
Free developer account. I tested this app and it was confirmed working on 13.3. Thanks for responding!

**Comment by jil-stripe** [comment]
@ClayMav,
I'm checking back in: it looks like the Alamofire folks are reporting that this was fixed with Xcode 11.4 and iOS 13.4. Could you update your Xcode and iOS versions and let me know if you're still unable to run the app? (Make sure to re-run the `setup.sh` script.)

Thanks!
James

**Comment by jil-stripe** [comment]
Going to close this issue. Feel free to reopen if you're still experiencing issues.

James
